src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +143,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
+    startPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the starting page',
     },
-    'ending-page': {
+    endPage: {
       validate: (val) => isNumeric(val),
       errorMsg: 'Please enter the end page',
     },
   };
 
   const error = {};
   _mapObject(form, (val, key) => {
-    const isValid = fieldsToCheck[key].validate(val);
+    const isValid = fieldsToCheck[key].validate('' + val);
```

Comment:
Yes, that makes sense. So this `validate` method is NOT part of the `validator` library. That is why I suggested that you move the string conversion to each validate function above. Example: for the `startPage` object, the `validate` method is `(val) => (isNumeric(val) && val > '0')`. You can convert `val` to string in that function if it needs to be converted.